cds_migrator_kit/rdm/migration/transform/xml_processing/quality/parsers.py

```diff
@@ -133,7 +133,7 @@ def _clean(value_to_clean):
         if value_to_clean is not None:
             try:
                 if var_type is str:
-                    return clean_str(value_to_clean, regex_format, req, transform)
```

Comment:
is this change needed? I wouldn't touch this parser since I would like to phase it out